apisix/plugins/log-rotate.lua

```diff
@@ -266,11 +271,25 @@ local function rotate_file(files, now_time, max_kept)
 end
 
 
+-- get the custom log file name from local_conf, if any
+local function get_custom_logfile_name()
+    local local_conf = core.config.local_conf()
+
+    local new_error_log = core.table.try_read_attr(local_conf, "nginx_config", "error_log")
+    local new_access_log = core.table.try_read_attr(local_conf, "nginx_config", "http", "access_log")
```

Comment:
Is `custom_error_log_name` or `error_log_name` better?
The same with access log

Reply:
`custom_error_log_filename` seems better